docs/csharp/language-reference/operators/toc.yml

```diff
@@ -4,6 +4,9 @@
   - name: Arithmetic operators
     href: arithmetic-operators.md
     displayName: ++, --, +, -, *, /, %
+  - name: Boolean logical operators
+    href: boolean-logical-operators.md
+    displayName: "&, |, ^, !"
```

Comment:
@BillWagner question about the `displayName` field (as far as I understand it's *one* string that is checked if it contains the search entry): do we want to use commas to separate symbols? Or white spaces are more preferred? Now, if I search for "+ -" in the TOC I don't get "Arithmetic operators" entry; the search for "+, -" picks up the overview article. Is there any data about what search strings are more popular?

Reply:
Here's what the docs say on the `displayName` feature:

> `displayName` (optional) - Alternate search terms for TOC filtering. An additional string value that doesn’t get displayed (yes, it's a poor name) but is searched, as well as `name`, during TOC filtering. For multiple values, use a comma separated list such as `displayName: batch, asynchonous, off-line, offline`

We don't have data on what symbols people search for (yet). My hypothesis is that most will search for one operator token.

Reply:
> My hypothesis is that most will search for one operator token

I think that's the case too. So, I was nit-picking about low traffic cases. Then, I will follow the docs guidance to "use a comma separated list". Thank you for the docs quote!